Unix/tests/pal/test_pal.cpp

```diff
@@ -2416,6 +2427,7 @@ NitsEndTest
 //
 //==============================================================================
 
+#if WCHAR_TESTS_DISABLED
```

Comment:
Is this right? Don't you want to DISABLE the test if WCHAR_TESTS_DISABLED, not ENABLE the test? I think you're reversed here.